setup.py

```diff
@@ -34,24 +31,17 @@
     + [os.path.join(kasdir, f) for f in kas_source_files]
 )
 
-libraries = []
-if IS_WINDOWS:
-    # Needed for generating UUIDs in tskit
-    libraries.append("Advapi32")
+libraries = ["Advapi32"] if IS_WINDOWS else []
 
 _tsinfer_module = Extension(
     "_tsinfer",
     sources=sources,
     extra_compile_args=["-std=c99"],
     libraries=libraries,
-    # Enable asserts by default.
     undef_macros=["NDEBUG"],
     include_dirs=includes + [numpy.get_include()],
 )
 
 setup(
-    # The package name along with all the other metadata is specified in setup.cfg
-    # However, GitHub's dependency graph can't see the package unless we put this here.
-    name="tsinfer",
```

Comment:
Has this issue been resolved on GitHub?

Reply:
https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems says it should be in pyproject.toml, so will remove.